horace_core/sqw/@sqw/display.m

```diff
@@ -1,4 +1,4 @@
-function display(w)
+function w=display(w)
```

Comment:
Why does this need to return? It should never change the object.

Reply:
explained below